onnxruntime/wasm/api.cc

```diff
@@ -112,7 +115,9 @@ OrtSessionOptions* OrtCreateSessionOptions(size_t graph_optimization_level,
   // Enable ORT CustomOps in onnxruntime-extensions
   RETURN_NULLPTR_IF_ERROR(EnableOrtCustomOps, session_options);
 #endif
-
+#if defined(USE_XNNPACK)
```

Comment:
in future, we may want to offer the developers ability to manually set/unset XNNPACK EP.

Probably a good idea to deal with EPs in a separated function than OrtCreateSessionOptions(), which already have too many parameters.

It can be a good time to do now, or when OPENGL EP is introduced.